fe/fe-core/src/main/java/org/apache/doris/catalog/MysqlTable.java

```diff
@@ -156,6 +211,8 @@ public int getSignature(int signatureVersion) {
         String charsetName = "UTF-8";
 
         try {
+            // resource name
+            adler32.update(externalCatalogResourceName.getBytes(charsetName));
             // name
```

Comment:
the following `host`, `port` maybe null. we need to check them or use `getxxx()` methods.